src/internal/connector/onedrive/item.go

```diff
@@ -186,17 +224,17 @@ func oneDriveItemInfo(di models.DriveItemable, itemSize int64) *details.OneDrive
 func oneDriveItemMetaInfo(
```

Comment:
nit: Should we change the name to suggest that we are only returning permissions now instead of entire metadata.